app/views/landing.scala.html

```diff
@@ -15,41 +15,59 @@
  *@
 
 @import config.AppConfig
-@import repositories.declaration.Submission
+@import models.Declaration
+@import models.DeclarationActionType.DeclarationActionType
 @import uk.gov.hmrc.play.views.formatting.Dates._
 @import org.joda.time.LocalDate
 
-@(submissions: Seq[Submission])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+@(submissions: Seq[Declaration])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
-    @main_template(title = messages("common.importDeclarations"), bodyClasses = None, mainClass = Some("full-width")) {
-        <div class="form-group">
-            <h1 class="heading-xlarge">@messages("common.importDeclarations")</h1>
-        </div>
-        @if(submissions.nonEmpty) {
-            <p><a class="button" href="@routes.DeclarationController.displaySubmitForm("declarant-details")">@messages("common.button.createNew")</a></p>
+@main_template(title = messages("landingpage.title"), bodyClasses = None, mainClass = Some("full-width")) {
```

Comment:
Don't need `bodyClasses` here as the default is `None`
```suggestion
@main_template(title = messages("landingpage.title"), mainClass = Some("full-width")) {
```

Reply:
That already existed, although I'm happy to remove it